### PR TITLE
Adding webpack and some updated

### PR DESCRIPTION
**Pull request by lwinkyawmyat**
## Motivation
- Add `webpack`
- Add new testing tool call `mocha` and remove old test toots call `test`
- Using es6 syntax in `lib/rabbit.js` 
- Updated `README.md` with new installation method

## Test Plan
Pass
Testing library

Reduce file size:
build result

### CODE REVIEW

**Comment by yelinaung** [comment]
@dreamingblackcat Mind help us take a look at this ?

File: lib/rabbit.js [hunk 1]
```diff
@@ -1,799 +1,800 @@
...
+    const from_regex = new RegExp(from,"g");
+    output = output.replace(from_regex, to);
   }
 
   return output;
 
 }
 
+var Rabbit = {
+  "zg2uni" : zg2uni,
```
Comment (lwinkyawmyat) [inline lib/rabbit.js]: Should be function instead of property.

File: lib/rabbit.js [hunk 2]
```diff
@@ -1,799 +1,800 @@
...
+  "uni2zg": uni2zg
```
Comment (lwinkyawmyat) [inline lib/rabbit.js]: Should be function instead of property.

File: test/rabbit.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const chai = require('chai');
+const Rabbit = require('./../index');
```
Comment (lwinkyawmyat) [inline test/rabbit.test.js]: Should be test with 'lib/rabbit.js` before building for production.

**Review by dreamingblackcat — COMMENTED** [review]
Nice update!

File: .travis.yml [hunk 4]
```diff
@@ -1,11 +1,10 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+  - "6"
```
Comment (dreamingblackcat) [inline .travis.yml:3]: :+1:  This should definitely be updated. Great catch!

**Comment by dreamingblackcat** [comment]
@lwinkyawmyat @yelinaung @Arkar-Aung  Sorry for taking so long to review it. Here are my opinions:

- Updating node versions (Definitely a must :+1:  )
- Adding a new test framework (Not sure if it's worth it because we are only using assert. And normally we would want to avoid unnecessary dependencies. However a test framework is a devDependencies so it doesn't hurt. I am okay with either way)
- Adding webpack (Definitely a lot of benefits by adding webpack :+1: :+1: )
- Inlining the tests ( I am fine either way whether it's inline or via a sample.json. There are definitely room for improvement here. For eg. we could break up each assertions into separate cases where we add description of exactly what specific conversion we are testing.)

Let me know your thoughts @yelinaung  @Arkar-Aung . The PR is green so should be able to merge it if you guys give a :+1: .

**Comment by yelinaung** [comment]
I've no clue about Javascript. I will leave it to @saturngod and @Arkar-Aung

**Comment by Arkar-Aung** [comment]
@yelinaung  @lwinkyawmyat It looks good and is a nice update as far as I check. :+1: Good to merge. Btw, :+1: to @dreamingblackcat for concrete review. Any comment on that @saturngod ?

**Comment by lwinkyawmyat** [comment]
Hi @Arkar-Aung, Can you update on [NPM](https://www.npmjs.com/package/rabbit-node)? 👍 👍 👍

**Pull request merged**